src/App.test.js

```diff
@@ -65,10 +67,12 @@ describe('App', () => {
 
   beforeEach(() => {
     wrapper = shallow(
+      <div>
       <Header searchFor={searchForMock} />
       <CardContainer parlors={parlors}
         icecream={icecream}
         search={value} />
+        </div>
     );
   })
```

Comment:
Things to test: 

Default states for our app.
That it rendered correctly? << May be unnecessary